nginx.conf

```diff
@@ -0,0 +1,35 @@
+worker_processes auto;
```

Comment:
Let's specify user under that nginx processes will work - https://nginx.org/en/docs/ngx_core_module.html#user

Reply:
Do I still need to specify the user, if I'm using unprivileged image of Nginx? I switch to user nginx in the dockerfile, before executing CMD nginx

Reply:
Yes, potentially we can run docker container as root user if we need so, even if it has an `nginx` default user, in such a case nginx with this config would be started by `root` user instead of `nginx` one